guides/common/modules/con_provisioning-templates.adoc

```diff
@@ -4,9 +4,9 @@
 A provisioning template defines the way {ProjectServer} installs an operating system on a host.
 
 {ProjectName} includes many template examples.
-In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning templates* to view them.
+In the {ProjectWebUI}, navigate to *Hosts* > *Templates* > *Provisioning Templates* to view them.
 You can create a template or clone a template and edit the clone.
-For help with templates, navigate to *Hosts* > *Provisioning templates* > *Create Template* > *Help*.
+For help with templates, navigate to *Hosts* > *Templates* > *Provisioning Templates* > *Create Template* > *Help*.
```

Comment:
I went through the WebUI navigation and my perspective is a bit different.

`Templates` is the header of the group of entries, and you click one of the entries? Then I would go without adding `> Templates`. (Applies Everywhere)
If you want to help the reader find the entry in the long list, you can append `(in the Templates section)` or something similar.

I also went through the style guide but did not get any supporting section for the header to be the part of sequence of selection. However, I am open to the suggestion if mistaken.

I am okay with every other thing though.

Reply:
You have to click "Hosts", then click "Templates", and then click "Provisioning Templates". To me, this means that it has to be documented as "..., navigate to *Hosts* > *Templates* > *Provisioning Templates*." So I'm 100% ACK with Adam's PR here.

Reply:
`Templates` is not clickable in WebUI.

Reply:
It _is_ clickable on Foreman 3.8.0. It does not open a page if you click it, but expand the menu.

Reply:
Yes, my snap was not showing click-to-expand. But, confirmed with SME :+1:

Reply:
You can also verify this by installing the latest satellite stream from ohsnap.